docs/en/wallet-attestation.rst

```diff
@@ -118,14 +118,31 @@ Wallet Instance Initialization and Registration
 
 **Step 9**: The Wallet Instance sends the ``challenge`` with Key Attestation and Cryptographic Hardware Key Tag to the Wallet Provider Backend in order to register the Wallet Instance identified with the Cryptographic Hardware Key public key.
 
-.. note::
+The request to the Wallet Provider endpoint in order to register the Wallet Instance MUST use the HTTP POST method with the parameters in the message body encoded in ``application/json`` format and the following parameters MUST be provided:
```

Comment:
```suggestion
In order to register the Wallet Instance, the request to the Wallet Provider MUST use the HTTP POST method. The parameters MUST be encoded in the `application/json` format and included in the message body. The following parameters MUST be provided:
```

I'd propose to give a name to the wallet registration endpoint in the wallet solution architecture